cmd/tailscale/ui.go

```diff
@@ -256,18 +304,24 @@ func (ui *UI) layout(gtx layout.Context, sysIns system.Insets, state *clientStat
 				if netmap == nil || state.backend.State < ipn.Stopped {
 					return D{}
 				}
+				for ui.self.Clicked() {
+					events = append(events, CopyEvent{Text: localAddr})
+					ui.showCopied(gtx, localAddr)
+				}
 				return ui.layoutLocal(gtx, sysIns, localName, localAddr)
 			case 2:
+				return ui.layoutExitStatus(gtx, &state.backend)
```

Comment:
I noticed in my testing that switching from a router back to "None" caused this panel to stick around for a few seconds more. In one case it wouldn't close at all.

Not sure if this is an IPN state issue, but calling it out in case you have thoughts.

Reply:
Fixed. I had mistakenly forgot to update the relay state when the preference change but not the netmap. For some reason, my test device always got a netmap update immediately after the preference update.